backend/src/main/java/com/google/sps/scraping/BBCGoodFoodRecipeScraper.java

```diff
@@ -53,6 +53,11 @@ public static Recipe scrapeRecipe(String url) {
     }
   }
 
+  /*Created this function to output  */
+  public static String searchRecipeLink(String ingredients, String key) {
+    return "https://customsearch.googleapis.com/customsearch/v1?cx=c318350d7878a8a31&exactTerms="
+        + ingredients +"&key=" + key;
+  }
```

Comment:
- Add newline after line 60
- I think the comment above the function is incomplete, I didn't quite get what the method does just by reading the comment. Can you please re-write?